### PR TITLE
Completed q1.js

### PR DESCRIPTION
**Pull request by kwu25**

### CODE REVIEW

File: kelvinWu/quiz/q1.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+////if index < k, add it to sum
+////else if we are >= k i.e. we have reached the k window size for the first k elements already, need to slide the window
+//////delete the number k away from currentIndex
+//////add currentNumber to index
+////after add adding and/or subtracting numbers, check if current subarraysum is greater than maxSum.
+//////replace maxSum with currentSum
+
+// k = 2
+// i:   0 1 2 3 4 5
+// arr: 1 2 3 4 5 6
```
Comment (knrodriguez) [inline kelvinWu/quiz/q1.js:23]: Good approach. Missing edge cases.

File: kelvinWu/quiz/q1.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    3
+  )}`
+);
+
+console.log(
+  `[${[1, 1, 1, 1, 1, 1]}]  k: ${6}  answer: ${maxSubarraySum(
+    [1, 1, 1, 1, 1, 1],
+    6
+  )}`
+);
```
Comment (knrodriguez) [inline kelvinWu/quiz/q1.js:67]: Missing tests that check for edge cases.

**Review by knrodriguez — APPROVED** [review]

**Comment by knrodriguez** [comment]
Nice job, but don't forget about edge and test cases!
